src/mol-plugin/behavior/dynamic/volume-streaming/util.ts

```diff
@@ -69,7 +69,8 @@ export async function getContourLevel(provider: 'emdb' | 'pdbe', plugin: PluginC
 export async function getContourLevelEmdb(plugin: PluginContext, taskCtx: RuntimeContext, emdbId: string) {
     const emdbHeaderServer = plugin.config.get(PluginConfig.VolumeStreaming.EmdbHeaderServer);
     const header = await plugin.fetch({ url: `${emdbHeaderServer}/${emdbId.toUpperCase()}/header/${emdbId.toLowerCase()}.xml`, type: 'xml' }).runInContext(taskCtx);
-    const contours = header.getElementsByTagName('contour');
+    const map = header.getElementsByTagName('map')[0];
+    const contours = map.getElementsByTagName('contour');
```

Comment:
So this wasn't working at all before?

Reply:
Only happens for header files which reference multiple primary contours (like https://ftp.wwpdb.org/pub/emdb/structures/EMD-11022/header/emd-11022.xml which has additional primary items as part of the `interpretation` tag).

Think that's quite rare and took us some time to run into the issue.

Reply:
So doesn't this break the existing use case? Or the format was that when there is just 1 map there is a global contour property, but the map array is always present?

Reply:
Ok, sounds good, makes sense. Thanks.